src/nsls2api/api/v1/admin_api.py

```diff
@@ -55,14 +55,8 @@ async def generate_user_apikey(username: str):
 
 @router.post("/admin/proposal/generate-test")
 async def generate_fake_proposal(
-    use_real_people: bool = False,
+    add_specific_user: str,
 ) -> Optional[SingleProposal]:
-    if use_real_people:
-        raise HTTPException(
-            status_code=fastapi.status.HTTP_501_NOT_IMPLEMENTED,
-            detail="Creating proposals using real people is not implemented yet",
-        )
-
     proposal = await proposal_service.generate_fake_test_proposal()
```

Comment:
Didn't notice this before, but should we pass `add_specific_user` string into this method?

Reply:
Addressed in [6ccd689](https://github.com/NSLS2/nsls2api/pull/97/commits/6ccd6899e9277a9ebfb79e7cc41925bb6b893c41) Let me know if this solution is specific, it currently passes through `None` for facility so it will default to NSLS2

Reply:
The other options would be to swap the order of the parameters for the `proposal_service.generate_fake_test_proposal()` method, or add the facility as another optional parameter to the endpoint. 

But I think for now this is ok.